### ISSUE TITLE
Archivist Fox role, maybe?

### ISSUE THREAD

**Issue by McTsts** [McTsts/Werewolves-Roles#527]
ooooh Fox Archivist, learns a role (or more) from the role list every day if they dont join :)

_Originally posted by @McTsts in https://github.com/McTsts/Werewolves-Roles/issues/461#issuecomment-1127944614_

**Comment by 12Metatron** [comment]
Would pack learn the roles as well?

**Comment by McTsts** [comment]
Should've copied this over as well:
> Maybe to fit with the general way the foxes work, the fox learns a role (or several) every day and the pack learns the alignment?

**Comment by 12Metatron** [comment]
Sounds like a fun role. A fox spin on knowledge wolf, so they slowly learn over time instead of all at once. Good idea overall

**Comment by McTsts** [comment]
What I'm unsure about is how much information they should learn. A single role doesn't really feel like very much?

**Comment by McTsts** [comment]
Actually, this might also solve the issue with what info is sent to the wolfpack: The fox learns 2 (or so) roles and may pick 1 that is sent to the wolfpack

**Comment by 12Metatron** [comment]
> What I'm unsure about is how much information they should learn. A single role doesn't really feel like very much?

This depends on if its every phase, or only day phase. Is it just day phase?

**Comment by McTsts** [comment]
The general way foxes work is they do something during the day, and then the wolves learn something at the start of the night

**Comment by 12Metatron** [comment]
> Actually, this might also solve the issue with what info is sent to the wolfpack (just the category is a bit pointless): The fox learns 2 (or so) roles and may pick 1 that is sent to the wolfpack

My opinion would be to leave the exact number at the hosts discretion, just like knowledge wolf. But maybe minimim 2.

**Comment by McTsts** [comment]
Yeah, could probably do more than 2, but I think the idea of the fox picking one of the roles they've learned and sending it to the pack could work. That way the pack learns something meaningful, but not as much as the fox

**Comment by Mjnstag** [comment]
Would the archivist Wolf get a say in the category they learn from? Also, would the role they learn be predetermined by host before game starts or randomly rolled? Personally i dont really like the random idea, but if its predetermined that would need to be balanced by hosts which could be problematic and difficult to do

**Comment by McTsts** [comment]
I assume the hosts would determine it (so like KW) - could possibly allow the selection of a category though which I think might be more fun to play.

**Comment by Shepherdwhen** [comment]
The information could be preset at the start of the game, set for like days of info = players/3 rounded. This means the host could have info appear at certain times in the game

**Comment by Shepherdwhen** [comment]
Example could be that D1 literally before Assassin can kill, wolves learn at least one TK is assassin

**Comment by venomousbirds** [comment]
> My main issue with AF is a host that decides to give it bad info because town made mistakes and wolves are winning, or good info because wolves are badly losing.

> The only way to circumvent it would to be to plan each day's worth of info beforehand, which would make it less encouraging to use

we plan kw lists ahead anyways, plus its just another arbitrary thing, no need to worry about thinking of stuff. Of course you won't give them the info that there's a huntress ingame when huntress has already been revealed but theres so much info you can give that does work even later in the game

**Comment by ThomasRubycrust** [comment]
If we do @Mjnstag suggestion of letting them pick category, it'd be hard to plan beforehand.

**Comment by venomousbirds** [comment]
then we probably shouldnt do that

> If we do @Mjnstag suggestion of letting them pick category, it'd be hard to plan beforehand.

**Comment by venomousbirds** [comment]
not letting them pick the same one twice would make it more managable for the hosts

**Comment by Shepherdwhen** [comment]
What if it gets a cryptic at the start of the game and it can select one piece of information to send to the pack + get the unencrypted info itself

**Comment by ThomasRubycrust** [comment]
What do you mean?

**Comment by ThomasRubycrust** [comment]
I think I prefer what I said. It's less complex and 100% consistent with the other foxes (all of them share a role with the pack)

**Comment by Shepherdwhen** [comment]
Another example

The Solo's effect may be lethal
Pyro/Nightmare/Plague

**Comment by Shepherdwhen** [comment]
> I think I prefer what I said. It's less complex and 100% consistent with the other foxes (all of them share a role with the pack)

I was thinking it could also share an answer with the pack but then that gives it no reason to join the pack

**Comment by cryptidjayden** [comment]
> What if it gets a cryptic at the start of the game and it can select one piece of information to send to the pack + get the unencrypted info itself

no. that's awful.
